R/derive_extreme_event.R

```diff
@@ -331,6 +476,7 @@
 #' ) %>%
 #'   filter(PARAMCD == "CBOR")
 #'
+
```

Comment:
Do we need a blank line here and above?

Also we don't have a example for `check_type` around handling duplicates here - i would consider linking to stefan's `derive_extreme_records()` example here? i did similar with `derive_vars_joined()` linking to `derive_vars_merged()`. These uniqueness checks do catch out users sometimes